doc/stages/readers.i3s.rst

```diff
@@ -0,0 +1,43 @@
+.. _readers.i3s:
+
+readers.i3s
+===========
+
+`Indexed 3d Scene Layer (I3S)`_ is a specification created by Esri as a format for their 3D Scene Layer and scene services. The I3S reader handles RESTful webservices in an I3S file structure/format.
+
+Example
+--------------------------------------------------------------------------------
+This example will download the Autzen dataset from the arcgis scene server, reproject it to another spatial reference, and output it to a las file. This is done through PDAL's command line interface or through the pipeline.
+
+.. code-block:: json
```

Comment:
Not sure it's useful to do reprojection stuff in this example since I3S is the focus.  Also, fix JSON - missing a comma.